developer-docs-site/docs/tutorials/testnet-devnet-incentivized-testnet.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Testnet, Devnet and Incentivized Testnet"
+slug: "testnet-devnet-and-incentivized-testnet"
+---
+
+# Testnet, Devnet and Incentivized Testnet
+
+You can connect to Aptos Blockchain in a few ways. See the below table:
+
+:::tip
+
+Make sure to see the row describing **What not to do**.
+
+:::
+
+|Description |Aptos Devnet | Testnet | Aptos Incentivized Testnet (AIT)|
+|---|---|---|---|
+|**Network runs where**| Runs on Aptos servers.| Runs locally on your computer, entirely independent from devnet. | Runs on Aptos servers.|
```

Comment:
This is not quite true:
- Aptos Devnet: Validators run on Aptos servers. Fullnodes are both run by Aptos and the public (i.e., the Aptos community).
- Local Testnet: Validators and Fullnodes run locally on your computer.
- Aptos Incentivized Testnet (AIT): Some Validators run on Aptos servers, others are community run. Fullnodes are both run by Aptos and the public (i.e., the community)